### PR TITLE
https://issues.redhat.com/browse/ACM-15310

### PR DESCRIPTION
**Pull request by dockerymick**

### CODE REVIEW

File: release_notes/acm_deprecate_remove.adoc [hunk 1]
```diff
@@ -40,6 +40,13 @@ A _deprecated_ component, feature, or service is supported, but no longer recomm
 | Enable the _Fleet view_ switch to view the new default _Overview_ page.
 | The previous layout of the {acm-short} _Overview_ page is deprecated.
 
+| Policy compliance history API
+| Governance 
+| 2.12
+| None
+//not really sure what the recommendation is
```
Comment (mprahl) [inline release_notes/acm_deprecate_remove.adoc]: A recommendation could be:

> Leverage the existing policy metrics to see compliance status changes and/or scrape config-policy-controller and cert-policy-controller pod logs to get a detailed compliance history per managed cluster.

**Comment by dockerymick** [comment]
@mprahl thanks for the review and feedback! Recently updated the file.

**Review by oafischer — APPROVED** [review]
lgtm

**Review by mprahl — APPROVED** [review]

**Comment by dockerymick** [comment]
@mprahl @oafischer thanks to you both for the reviews and approvals. Merging this PR

**Pull request merged**